site/learn/expert-advice/identity-basics/common-authentication-implementation-risks.md

```diff
@@ -0,0 +1,122 @@
+
+How are you going to authenticate requests coming from other platforms? How will other platforms validate the requests you make to them? You'll need to implement standardized auth protocols like OAuth, OpenID, and SAML to perform such third-party integrations.
+
+If your company has the time, money, and people to invest, and considers authentication as a core business domain/differentiator, then implementing these kinds of additional security features might be the best approach. But for the majority of us, that's not the case. A third-party solution can typically get you these more advanced features out of the box.
+
+## Vendor Assessment
+
+If you're considering using a third-party vendor, then you need to make sure you perform due diligence in [assessing how appropriate and risky that vendor is](/learn/expert-advice/identity-basics/due-diligence-authentication-vendors/). Many regulatory programs like SOC2 require some kind of controls around vendor risk management.
+
+While this article has already touched on areas of risk in implementing an authsystem, like application security, financials, and performance, assessing a vendor is where the rubber meets the road. It's where you apply all your knowledge about risk in authentication implementation.
+
+For example, you'll want to consider how coupled you'll become to a vendor's solution (otherwise known as [vendor lock-in](https://www.factioninc.com/blog/vendor-lock-in/)). You may want the freedom to replace a vendor in the future, perhaps having found a better auth service that's cheaper or supports a compliance program your current vendor does not. Does the vendor you are considering now make it easy or difficult to switch?
+
+Broadly speaking, make sure you answer each of these questions before deciding to move forward with a vendor:
+
+- Does this vendor meet required security standards like SOC2, HIPAA, or COPPA?
+- Does this vendor perform regular penetration testing?
+- Is there any risk of this vendor and its services affecting my organization's reputation?
+- Is the cost of the vendor's service a potential financial risk?
+- How does the vendor calculate pricing? Is it usage based, a flat fee, or something else?
+- If our organization ever wanted to switch vendors, does this vendor offer some kind of data export functionality?
+
+Of course, this is just a starting point. These questions bring up other fundamental questions like, "What is our threshold for considering a particular vendor's service as risky?" Your organization will have to answer that, and other subjective questions, based on your own risk tolerance, business agility, and company culture.
+
+## Conclusion
+
+Authentication is not simple. In this article alone, you've seen just how many areas affect and are affected by authentication: security, your organization's reputation, financials, project timelines, governance, and so much more.
```

Comment:
Is this the right usage of affect vs effect?

Reply:
Yup. Affect is the verb, effect is the noun.